docs/toml-schema.md

```diff
@@ -133,6 +139,14 @@ zulip-stream = "t-lang"
 # Default is 0.
 weight = -100
 
+# Customized roles held by a subset of the team's members, beyond "Team leader"
+# which is rendered automatically for members of the `leads` array.
+[[website.roles]]
```

Comment:
Namespacing these roles under website feels a bit strange. I know the reason for this PR is to display this information on the website, but is this information only relevant to the website? Perhaps this should just be a top level `roles` array?

Reply:
Fixed.